site/src/api/typesGenerated.ts

```diff
@@ -1079,6 +1083,10 @@ export const Entitlements: Entitlement[] = [
   "not_entitled",
 ]
 
+// From codersdk/experiments.go
+export type Experiment = "vscode_local"
+export const Experiments: Experiment[] = ["vscode_local"]
```

Comment:
Is this ok that the const shares a name with a type?

Reply:
IIRC it's valid, but can be confusing.

Reply:
Actually, this is a common thing in our types generated. I see the PR that introduced to prevent duplicating lists on the FE: https://github.com/coder/coder/pull/5734

It makes a lot of sense, just unfortunate we collide with names a lot. For this PR it is irrelevant, it should be a change to the generator if we decide to fix it.